src/libtorch.cc

```diff
@@ -651,6 +651,10 @@ ModelInstanceState::ValidateTypedSequenceControl(
   return nullptr;  // success
 }
 
+// The naming convention followed for inputs/outputs in the model configuration.
+// Outputs don't support ARGUEMENT.
+typedef enum { INDEX, ARGUEMENT, STRICT_ORDERING } NamingConvention;
```

Comment:
NamingConvention -> IOTensorOrder

INDEX, FORWARD, CONFIG??? 

It makes sens as we look at the name of the tensors to identify the order in which we must place the arguments.

Reply:
Use C++ style enum